addons/project/static/src/js/project_kanban.js

```diff
@@ -77,6 +106,24 @@ const ProjectTaskKanbanRenderer = KanbanRenderer.extend({
         KanbanColumn: ProjectTaskKanbanColumn,
     }),
 
+    init: function () {
+        this._super.apply(this, arguments);
+        this.isProjectManager = false;
+    },
+
+    willStart: function () {
+        const self = this;
+        const superPromise = this._super.apply(this, arguments);
+
+        const isProjectManager = this.getSession().user_has_group('project.group_project_manager').then(function (hasGroup) {
+            self.isProjectManager = hasGroup;
+            self._setState();
+            return Promise.resolve();
+        });
```

Comment:
We could use arrow functions 🙂 
```suggestion
        const superPromise = this._super.apply(this, arguments);

        const isProjectManager = this.getSession().user_has_group('project.group_project_manager').then(hasGroup  => {
            this.isProjectManager = hasGroup;
            this._setState();
            return Promise.resolve();
        });
```